app/main.py

```diff
@@ -1,3 +1,52 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            km = other.km
+        elif isinstance(other, (int, float)):
+            km = other
+        return Distance(self.km + km)
+
+    def __iadd__(self, other):
+        if isinstance(other, Distance):
+            self.km += other.km
+        elif isinstance(other, (int, float)):
+            self.km += other
+        return self
+
+    def __mul__(self, other):
+        return Distance(self.km * other)
+
+    def __truediv__(self, other):
+        return Distance(round((self.km / other), 2))
+
+    def __lt__(self, other):
+        if isinstance(other, (Distance, int, float)):
+            return self.km < other
+
+    def __gt__(self, other):
+        if isinstance(other, (Distance, int, float)):
+            return self.km > other
```

Comment:
if `other` is a `Distance` instance then the condition should be different:
```suggestion
            return self.km > other.km
```